### PR TITLE
chore: Update Dataset documentation and remove unused dependencies.

### PR DESCRIPTION
**Pull request by crismunoz**

### CODE REVIEW

File: docs/source/getting_started/explainability/metrics/spread.rst
Comment (fracarfer5) [inline docs/source/getting_started/explainability/metrics/spread.rst:1]: I think the character "\" should be removed after :math: is used, is giving a rendering error

File: docs/source/getting_started/explainability/metrics/spread.rst [hunk 1]
```diff
@@ -73,7 +73,7 @@ Methodology
    - where \( D_{KL} \) is the Kullback-Leibler divergence, \( P \) is the normalized feature importance distribution, \( Q \) is the uniform distribution, and \( M = \frac{1}{2}(P + Q) \).
```
Comment (fracarfer5) [inline docs/source/getting_started/explainability/metrics/spread.rst]: missing :math: in inline equations

Comment (crismunoz) [reply docs/source/getting_started/explainability/metrics/spread.rst]: done

File: docs/source/getting_started/explainability/metrics/spread.rst
Comment (crismunoz) [reply docs/source/getting_started/explainability/metrics/spread.rst:1]: is necessary

**Review by fracarfer5 — COMMENTED** [review]
For me is ok

**Pull request merged**